src/SFML/Graphics/GLExtensions.hpp

```diff
@@ -34,80 +34,175 @@
-    #define GLEXT_GL_DEPTH_COMPONENT               GL_DEPTH_COMPONENT16_OES
-    #define GLEXT_GL_INVALID_FRAMEBUFFER_OPERATION GL_INVALID_FRAMEBUFFER_OPERATION_OES
-    #define GLEXT_texture_non_power_of_two         false
-    #define GLEXT_multitexture                     true
-    #define GLEXT_glClientActiveTexture            glClientActiveTexture
-    #define GLEXT_glActiveTexture                  glActiveTexture
-    #define GLEXT_GL_TEXTURE0                      GL_TEXTURE0
-    #define GLEXT_glBlendEquation                  glBlendEquationOES
-    #define GLEXT_GL_FUNC_ADD                      GL_FUNC_ADD_OES
-    #define GLEXT_GL_FUNC_SUBTRACT                 GL_FUNC_SUBTRACT_OES
+    #define GLEXT_glBlendFuncSeparate                 glBlendFuncSeparateOES
 
+    // Core since 2.0 - OES_blend_equation_separate
+#ifdef SFML_SYSTEM_ANDROID
+    // Hack to make transparency working on some Android devices
+    #define GLEXT_blend_equation_separate             false
 #else
+    #define GLEXT_blend_equation_separate             GL_OES_blend_equation_separate
+#endif
+    #define GLEXT_glBlendEquationSeparate             glBlendEquationSeparateOES
 
-    #include <GL/glew.h>
-    #include <SFML/OpenGL.hpp>
+    // Core since 2.0 - OES_texture_npot
+    #define GLEXT_texture_non_power_of_two            GL_OES_texture_npot
+
+    // Core since 2.0 - OES_framebuffer_object
+    #define GLEXT_framebuffer_object                  GL_OES_framebuffer_object
+    #define GLEXT_glBindRenderbuffer                  glBindRenderbufferOES
+    #define GLEXT_glDeleteRenderbuffers               glDeleteRenderbuffersOES
+    #define GLEXT_glGenRenderbuffers                  glGenRenderbuffersOES
+    #define GLEXT_glRenderbufferStorage               glRenderbufferStorageOES
+    #define GLEXT_glBindFramebuffer                   glBindFramebufferOES
+    #define GLEXT_glDeleteFramebuffers                glDeleteFramebuffersOES
+    #define GLEXT_glGenFramebuffers                   glGenFramebuffersOES
+    #define GLEXT_glCheckFramebufferStatus            glCheckFramebufferStatusOES
+    #define GLEXT_glFramebufferTexture2D              glFramebufferTexture2DOES
+    #define GLEXT_glFramebufferRenderbuffer           glFramebufferRenderbufferOES
+    #define GLEXT_GL_FRAMEBUFFER                      GL_FRAMEBUFFER_OES
+    #define GLEXT_GL_RENDERBUFFER                     GL_RENDERBUFFER_OES
+    #define GLEXT_GL_DEPTH_COMPONENT                  GL_DEPTH_COMPONENT16_OES
+    #define GLEXT_GL_COLOR_ATTACHMENT0                GL_COLOR_ATTACHMENT0_OES
+    #define GLEXT_GL_DEPTH_ATTACHMENT                 GL_DEPTH_ATTACHMENT_OES
+    #define GLEXT_GL_FRAMEBUFFER_COMPLETE             GL_FRAMEBUFFER_COMPLETE_OES
+    #define GLEXT_GL_FRAMEBUFFER_BINDING              GL_FRAMEBUFFER_BINDING_OES
+    #define GLEXT_GL_INVALID_FRAMEBUFFER_OPERATION    GL_INVALID_FRAMEBUFFER_OPERATION_OES
+
+#else
```

Comment:
With nested `#if/else` I'd recommend adding a comment mentioning line 33 or SFML_OPENGL_ES being not set.